src/app.ts

```diff
@@ -54,6 +55,13 @@ import getErrorCodeFromResponse from "./utils/getErrorCodeFromResponse";
 import { User } from "./types/user";
 import { toExpressHandler } from "./utils/express";
 
+const defaultModule = {
```

Comment:
why this is needed?

Reply:
Is needed for testing reason, for example to mock functions that are use only inside the `app.js` file like [here](https://github.com/teamdigitale/io-backend/pull/436/files#diff-f98485da1849ae631ba5c04e5cec07ddR191).